Flick/Cells/ListTableViewCell.swift

```diff
@@ -144,17 +155,36 @@ extension ListTableViewCell: UICollectionViewDataSource {
 
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
         // If list is empty, show 4 filler cells
-        return media.count == 0 ? 4 : media.count
+        if media.isEmpty {
+            return 4
+        } else {
+            return media.count == 10 ? media.count + 1 : media.count
+        }
     }
 
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        // TODO: Add left padding to first cell
-        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: mediaCellReuseIdentifier, for: indexPath) as? MediaInListCollectionViewCell else { return UICollectionViewCell() }
-        if media.count != 0 {
-            let media = self.media[indexPath.row]
-            cell.configure(media: media)
+        if indexPath.item == 10 {
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: seeAllCellReuseIdentifier, for: indexPath)
+            cell.backgroundColor = .lightGray2
```

Comment:
ooo I like this haha. very succinct. we can think about moving this into a helper function but this also isn't too complex so we don't absolutely have to